Jenkinsfile-deploy

```diff
@@ -0,0 +1,27 @@
+library("tdr-jenkinslib")
+
+pipeline {
+    agent {
+        label "master"
+    }
+    parameters {
+        choice(name: "STAGE", choices: ["intg", "staging", "prod"], description: "The stage you are building the api update lambda for")
+        string(name: "TO_DEPLOY", description: "The git tag, branch or commit reference to deploy, e.g. '1'")
+    }
+    stages {
+        stage("Deploy lambda") {
+            agent {
+                ecs {
+                    inheritFrom "aws"
+                    taskrole "arn:aws:iam::${env.MANAGEMENT_ACCOUNT}:role/TDRJenkinsNodeLambdaRole${params.STAGE.capitalize()}"
+                }
+            }
+            steps {
+                script {
+                    def accountNumber = tdr.getAccountNumberFromStage()
```

Comment:
Missing the stage param for the `tdr.getAccountNumberFromStage()` function

Reply:
One of these days I will remember this :smile: